src/components/geometry.js

```diff
@@ -35,6 +41,39 @@ module.exports.Component = registerComponent('geometry', {
 
     // Create new geometry.
     this.geometry = mesh.geometry = system.getOrCreateGeometry(data);
+    if (diffData.mergeTo) {
+      if (!data.mergeTo) { return; }
+      this.mergeTo(data.mergeTo);
+    }
+  },
+
+  mergeTo: function (toEl) {
+    var el = this.el;
+    var mesh = this.el.getObject3D('mesh');
+    var toMesh = toEl.getObject3D('mesh');
+    if (!toEl) {
+      warn('There is not a valid entity to merge the geometry to');
+      return;
+    }
+    if (toEl === el) { return; }
+    if (!toMesh) {
+      warn('Merge is not posible. The entity ' + toEl.id + ' doesn\'t have a mesh');
+      return;
+    }
+
+    if (toMesh.geometry instanceof THREE.Geometry === false) {
+      warn('Geometry merge is only available for THREE.Geometry types');
```

Comment:
Is it valid to have a  `mesh.geometry` that isn't an instance of THREE.Geometry?